.travis.yml

```diff
@@ -3,13 +3,25 @@ branches:
     - master
 language: python
 python:
-  - "3.4"
+- '3.4'
 before_script:
-  - pip install pygithub
-  - pip install pygithub3
+- if [ ${TRAVIS_BRANCH} = "master" ]; then [ -z "${GH_TOKEN}" ] && exit 0; [ ${TRAVIS_PULL_REQUEST}
```

Comment:
Quotation marks around `${GH_TOKEN}` can be removed as well (works in my project using similar config).